File: RMS/rms-service/src/main/java/com/maven/rms/models/OTCBalRC.java
package com.maven.rms.models;

import java.math.BigDecimal;
import java.math.BigInteger;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class OTCBalRC {
    
    private String coll_slip_no;
    private String orn_no;
    private String rcpt_no;
    private BigDecimal totalAmount;
    private String otc_pymt_mode;
    private String requested_by;
    private String approved_by;
    private String remark;
    private BigInteger mtt_id;
    private Integer otc_id;
    private Integer otc_counter_id;
    private String counter_id;
}

File: RMS-BackOfficePortal/src/app/core/models/otc-bal-rc.ts
export interface OTCBalRC{
    coll_slip_no: string;
    orn_no: string;
    rcpt_no: string;
    totalAmount: number;
    otc_pymt_mode: string;
    requested_by: string;
    approved_by: string;
    remark: string;
    mtt_id: number;
    otc_id: number;
    otc_counter_id: number;
    counter_id: string;
}